.github/workflows/dependent-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Install dependencies
       run: |
         pip install --upgrade pip
-        pip install --upgrade git+https://github.com/aws/amazon-braket-schemas-python@main
+        pip install --upgrade git+https://github.com/aws/amazon-braket-schemas-python
```

Comment:
```suggestion
        pip install --upgrade git+https://github.com/aws/amazon-braket-schemas-python@main
```

Reply:
I actually think we can remove this line all together, since there's no need here to install from github

Reply:
(we'll see if the dependent tests pass lol)